context-js/src/lib.rs

```diff
@@ -8,7 +8,10 @@ use wasm_bindgen::prelude::*;
 use wasm_streams::{readable::sys, readable::ReadableStream};
 
 #[wasm_bindgen]
-pub fn env_parse(env_vars: js_sys::Object) -> Option<env::parser::CIInfo> {
+pub fn env_parse(
+    env_vars: js_sys::Object,
+    stable_branches: Option<Vec<String>>,
```

Comment:
does this even need to be optional if we update bindings + add this param to the method in the same pr (in the monorepo)?